### PR TITLE
v50.2.0 RNs

### PR DESCRIPTION
**Pull request by jshackell-sfdc**

### CODE REVIEW

File: releasenotes/README.md [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+
+* CHANGE: The following org shape commands are now beta; they were pilot in v49.
+
+  * `force:org:shape:create`
+  * `force:org:shape:delete`
+  * `force:org:shape:list`
+
+* FIX: These release notes include the version of the CLI npm package (`sfdx-cli`) released each week, such as 7.77.1. ([GitHub issue #683](../../../issues/683))
+
+## 50.1.1 (October 17, 2020)
```
Comment (amphro) [inline releasenotes/README.md]: Can we add the CLI version here too @jshackell-sfdc

Comment (jshackell-sfdc) [reply releasenotes/README.md]: yep.

File: releasenotes/README.md [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+    
+     As we broke out the commands, we took the opportunity to also refactor the code. We’re keeping the current `force:auth:*` commands based on the original code. Test out the new commands and let us know if they behave differently or unexpectedly. We plan to alias the `force:auth:*` commands to the new `auth:*` commands in the future. Update your CI/CD scripts and let us know if you run into any issues with the new commands.
+
+* CHANGE: The following org shape commands are now beta; they were pilot in v49.
+
+  * `force:org:shape:create`
+  * `force:org:shape:delete`
+  * `force:org:shape:list`
+
+* FIX: These release notes include the version of the CLI npm package (`sfdx-cli`) released each week, such as 7.77.1. ([GitHub issue #683](../../../issues/683))
```
Comment (ekapner) [inline releasenotes/README.md]: Normally we avoid using "now" - but in this case, I would.

These release notes **now** include the version of the ...blah blah (not bolded in the notes, just here so you see it, ha)

**Review by ekapner — COMMENTED** [review]
I really approve but added one super minor nit

File: releasenotes/README.md (hunk 2 above)
Comment (jshackell-sfdc) [reply releasenotes/README.md]: Funny, I was originally going to put "now", then didn't, then did, then didn't... :). I think it's better with it. Thx.

**Review by ekapner — APPROVED** [review]

**Review by mdonnalley — APPROVED** [review]

**Pull request merged**